src/interactions/pointer.js

```diff
@@ -139,7 +143,7 @@ function pointerK(kx, ky, {x, y, px, py, maxRadius = 40, channels, render, ...op
           const rj = dx * dx + dy * dy;
           if (rj <= ri) (ii = j), (ri = rj);
         }
-        update(ii, ri);
+        update(ii, Math.hypot(px(ii) - xp, py(ii) - yp));
```

Comment:
Also, `ii` can be null here, and in that case we shouldn’t attempt to compute the distance.